src/main/mulmo/handler_graphai.ts

```
@@ -1,104 +1,8 @@
-    return {
-      data,
-      content: data.textContent,
-      hasNext: true,
-    };
-  } catch (e: unknown) {
-    console.log(JSON.stringify({ ok: false, url, error: String(e?.message || e) }));
-    return {
-      content: String(e?.message || e),
-    };
-  }
+  return await puppeteerCrawlerAgent.agent({namedInputs: { url }});
```

ESLint

[error] Replace `namedInputs:·{·url·}` with `·namedInputs:·{·url·}·`

(prettier/prettier)